src/Interpreters/InterpreterSelectQuery.cpp

```diff
@@ -2012,7 +2017,7 @@ void InterpreterSelectQuery::addPrewhereAliasActions()
         }
 
         auto syntax_result
-            = TreeRewriter(context).analyze(required_columns_all_expr, required_columns_after_prewhere, storage, storage_snapshot);
+            = TreeRewriter(context).analyze(required_columns_all_expr, required_columns_after_prewhere, storage, storage_snapshot, options.is_create_parameterized_view);
```

Comment:
Same comment as above -- probably it is enough to put this flag only in ASTQueryParameter and avoid passing it in the method arg?

Reply:
As mentioned in above comment, it would create the same crutch.